cmd/nerdctl/images.go

```diff
@@ -103,13 +104,43 @@ func imagesAction(cmd *cobra.Command, args []string) error {
 	var (
 		imageStore = client.ImageService()
 	)
+	var imageList []images.Image
 
-	// To-do: Add support for --filter.
-	imageList, err := imageStore.List(ctx, filters...)
-	if err != nil {
-		return err
-	}
+	if !cmd.Flags().Changed("filter") {
+		imageList, err = imageStore.List(ctx, filters...)
+		if err != nil {
+			return err
+		}
+	} else {
```

Comment:
I think this is still not supported right?
```
➜  nerdctl git:(rmd) ✗ docker images --filter "since=python:latest"
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
nginx        latest    b692a91e4e15   2 weeks ago   142MB
busybox      latest    7a80323521cc   2 weeks ago   1.24MB
```

```
➜  nerdctl git:(rmd) ✗ docker images --filter "since=python:latest" nginx
REPOSITORY   TAG       IMAGE ID       CREATED       SIZE
nginx        latest    b692a91e4e15   2 weeks ago   142MB
➜  nerdctl git:(rmd) ✗ 
```

Reply:
If you add some image names behind the images command, the nerdctl will filter the images name with the content in --filters.

`nerdctl images --filter "since=python:latest" nginx` means I want to get the image which is created before python:latest and name is nginx:latest

Reply:
Can we add a test for completeness?

Reply:
Sure